bin/cisc_to_adie_conversion.py

```diff
@@ -0,0 +1,153 @@
+    # construct full CISC ID (with sub-)
+    cid_full = 'sub-'+cid
+    for root,dirs,files in os.walk(sub):
+        for f in files:
+            # Only continue with the file if it's NOT a stupid annoying mac file
+            if 'DS_Store' not in f:
+                # Construct full path 
+                fullf = os.path.join(root,f)
+                # Create new path by replacing sub-CISCID with sub- ADIEID
+                newf = fullf.replace(cid_full,newid)
+                # Check that directories already exist - should've been created in F3 
+                if not os.path.isdir(os.path.split(newf)[0]):
+                    print("WARNING: '{}' does NOT exists".format(os.path.split(newf)[0]))
+                # If directory does exist, proceed with move and rename    
+                else:
+                    print("Attempting to move",fullf,"to",newf,'\n')
+                    shutil.move(fullf,newf)
+            elif 'DS_Store' in f:
+                continue
+
+# Function 5 - Count number of files in a sub- dir 
+def numfiles(subdr):
+    # initialize variable
+    numf = 0
+    for root, dirs, files in os.walk(subdr):
+        numf += len(files)
+        return numf
+    print ("Number of files in {}: {}".format(subdr,numf))
+
+# Function 6 - remove redundent dirs
+def compare(numf_old,numf_new,dir_to_del):
+    if numf_old == numf_new:
+        print ("Removing:",sub, '\n', 'Number of files equal - suggests successful conversion')
+        shutil.rmtree(dir_to_del)
+    else:
+        print (" Not removing:",sub,'\n','Number of files not equal!')
+
+
+# TODO: F7 - Chnage participants.tsv 
+
+
+
+# -------------- RUN FUNCTIONS -------------- #
+# This bit will be in /test/ directory 
+# For now, just loop over one subject for testing purposes 
+# sourcery skip
+subdirs = os.path.join(adie_dir, 'wills_data/test_data/sub-*')
+
+# Loop through each subject 
+for sub in glob.glob(subdirs):
+    # F5 - Store number of files in original directory 
+    numf_old = numfiles(sub)
+    # Get dict 
+    rename = convert_dict(txtfile)
+    # F1 - List sub directory for searching  
+    r = subpaths(sub)
+    # F2 - Extract CISC ID from root name, and match with ADIE ID 
+    # If CISC ID not recognized, return to start of loop and processnext subject  
+    try:
+        newid,cid = idmatch(r)
+        print(newid,cid)
+    except:
+        print ("Error occured! Ignoring", sub)
+        continue
+    # F3 - Recreate directory sturcture, using new ID names
+    newdir(newid,r)
+    # F4 - Copy files from old to new structure, while renaming
+    movefiles(sub,newid,cid)
+    # F5 - Store number of files in new directory 
+    numf_new = numfiles(os.path.join(os.path.dirname(sub),newid))
+    # F6 - Check if number of files in new sub dir is == to num files in original sub dir
+    compare(numf_old,numf_new,sub)
+    # Look prettier
+    print ('\n','#####------------------- NEW SUBJECT DIRECTORY --------------------#####','\n')
+
```

Comment:
Can you remove the trailing lines?

Reply:
Sorry what do u mean exactly by trailing lines in this case?